src/components/NightModeToggle/index.js

```diff
@@ -1,21 +1,34 @@
 /* global document */
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
 class NightModeToggle extends Component {
-  state = {
-    isNightMode: false,
-  };
+  static propTypes = {
+    isNightMode: PropTypes.bool.isRequired,
+    onToggle: PropTypes.func.isRequired
+  }
 
   toggleNightMode = () => {
-    document.body.classList.toggle('night-mode');
+    const { isNightMode, onToggle } = this.props;
+
+    if (isNightMode) {
+      document.body.classList.remove('night-mode');
+    } else {
+      document.body.classList.add('night-mode');
+    }
 
-    this.setState({ isNightMode: !this.state.isNightMode });
+    onToggle({ isNightMode: !isNightMode });
   }
 
   render() {
+    const { isNightMode } = this.props;
+
+    if (__CLIENT__ && isNightMode) {
```

Comment:
Probably want to just add `document.body.classList.add('night-mode');` inside of `componentDidMount(){}` if `isNightMode`. Putting it in the render will cause `document.body.classList.add('night-mode');` to continue to be called on every render